### ISSUE TITLE
How to allow login from specific domain emails?

### ISSUE THREAD

**Issue by juliusoh** [MomenSherif/react-oauth#2]

**Comment by MomenSherif** [comment]
According to google documentation it was only provided is Authorization flow only, you can use it like that using token/code flows,

```jsx
 const login = useGoogleLogin({
    onSuccess: resp => console.log(resp),
    hosted_domain: 'valeo.com',
  });
```

But It works also for `personalized button` & `one tap`, for some reason it was not mentioned in their [docs](https://developers.google.com/identity/gsi/web/reference/js-reference#IdConfiguration) 😅.

If you are using JS add `hosted_domain` prop

```jsx
    <GoogleLogin
        onSuccess={credentialResponse => {
          console.log(credentialResponse);
        }}
        hosted_domain="example.com" // add your domain
    />

```

if you are using typescript you will have type error, but I will fix it today and publish new version 0.2.1 tonight.

**Comment by MomenSherif** [comment]
[v0.2.1](https://github.com/MomenSherif/react-oauth/releases/tag/v0.2.1) Released! 🎉
